packages/account/src/Components/terms-of-use/terms-of-use-messages.tsx

```diff
@@ -5,7 +5,7 @@ import { Text } from '@deriv/components';
 
 export const Hr = () => <div className='terms-of-use__hr' />;
 
-export const BrokerSpecificMessage = ({ target }) => (
+export const BrokerSpecificMessage = ({ target }: { target: string }) => (
```

Comment:
as we know the possible values of target, can we try writing an enum for it with just these possible 5 values?

Reply:
Will get confirmation on the active broker codes then change to enum